plugin/manager.go

```diff
@@ -141,6 +150,24 @@ func (pm *Manager) reload() error {
 				return
 			}
 
+			if p.Rootfs != "" {
+				p.Rootfs = filepath.Join(pm.libRoot, p.PluginObj.ID, "rootfs")
+			}
+
+			// We should only enable rootfs propagation for certain plugin types that need it.
+			for _, typ := range p.PluginObj.Config.Interface.Types {
+				if typ.Capability == "volumedriver" && typ.Prefix == "docker" && strings.HasPrefix(typ.Version, "1.") {
+					if p.PluginObj.Config.PropagatedMount != "" {
+						// TODO: sanitize PropagatedMount and prevent breakout
+						p.PropagatedMount = filepath.Join(p.LibRoot, p.PluginObj.Config.PropagatedMount)
```

Comment:
@tiborvass : You should be using `filepath.Join(p.LibRoot, p.PluginObj.ID, p.PluginObj.Config.PropagatedMount)`. Else mounts from different plugins with the same name will collide. Another option is to fix `p.LibRoot` to include the pluginID (It should have been this way in the first place)

We dont want to use hard coded `mnt`. We want plugin devs to specify the mount that needs propagation.